conda_smithy/templates/travis.yml.tmpl

```diff
@@ -1,5 +1,5 @@
 
-{% set conda_vars = {'python': 'CONDA_PY', 'numpy': 'CONDA_NPY'} %}
+{% set conda_vars = {'python': ' CONDA_PY', 'numpy': 'CONDA_NPY'} %}
```

Comment:
Without the space travis was not correctly identifying the build matrix.

[build #9] (https://travis-ci.org/scikit-xray/scikit-xray-feedstock-dev/builds/82626440) Without space there is only one build
[build #10] (https://travis-ci.org/scikit-xray/scikit-xray-feedstock-dev/builds/82627095) With the space there are two builds
[diff between build #10 and build #9] (https://github.com/scikit-xray/scikit-xray-feedstock-dev/compare/ecc4e78a2966...975967fe6c9f)